cmd/notation/policy/import.go

```diff
@@ -0,0 +1,82 @@
+package policy
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation-go/verifier/trustpolicy"
+	"github.com/notaryproject/notation/cmd/notation/internal/cmdutil"
+	"github.com/notaryproject/notation/internal/cmd"
+	"github.com/notaryproject/notation/internal/osutil"
+	"github.com/spf13/cobra"
+)
+
+type importOpts struct {
+	cmd.LoggingFlagOpts
+	filePath string
+	force    bool
+}
+
+func importCmd() *cobra.Command {
+	var opts importOpts
+	command := &cobra.Command{
+		Use:   "import [flags] <file_path>",
+		Short: "[Preview] Import trust policy configuration from a JSON file",
+		Long: `[Preview] Import trust policy configuration from a JSON file.
+
+** This command is in preview and under development. **
+
+Example - Import trust policy configuration from a file:
+  notation policy import my_policy.json
+`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.filePath = args[0]
+			return runImport(cmd, opts)
+		},
+	}
+	opts.LoggingFlagOpts.ApplyFlags(command.Flags())
+	command.Flags().BoolVar(&opts.force, "force", false, "override the existing trust policy configuration, never prompt")
+	return command
+}
+
+func runImport(command *cobra.Command, opts importOpts) error {
+	// optional confirmation
+	policyPath := filepath.Join(dir.UserConfigDir, dir.PathTrustPolicy)
```

Comment:
`policyPath` is used in line 74, we could define it right above its usage.
Also, we should use `dir.ConfigFS()`:
```suggestion
	policyPath, err := dir.ConfigFS().SysPath(dir.PathTrustPolicy)
```